package-lock.json

```diff
@@ -244,7 +244,7 @@
       "integrity": "sha512-erl8P4PwU+1jm/ah8PBhW6UE1MkNkzSTRJTMnFFewzVp4rOtrVNAhKbf4eGQ+52qoim3gN5aXVjHpexqy75/gg==",
       "dev": true,
       "requires": {
-        "@types/node": "9.4.6",
+        "@types/node": "9.4.7",
```

Comment:
lol I guess this validation script worked since [it failed](https://travis-ci.org/edx/front-end-cookie-cutter-application/builds/353072805) without this change

![alt-text](https://media.giphy.com/media/LBA8IfDSb7TBS/giphy.gif)